install-cloud.sh

```diff
@@ -11,6 +11,14 @@
 
 set -e
 
+if [ -f /etc/centos-release ]
```

Comment:
Should we check whether these have been installed yet instead of just checking if it is centos?